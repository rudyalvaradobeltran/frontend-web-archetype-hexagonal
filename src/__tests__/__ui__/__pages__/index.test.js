import React from 'react';
import { create } from 'react-test-renderer';

// Mocks
import MockProvider from '../../../__mocks__/mockProvider';
// Components
import App from '../../../ui/pages/';

describe('PAGES', () => {
  it('Should render snapshot without crashing', () => {
    const wrapper = create(
      <MockProvider>
        <App />
      </MockProvider>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
