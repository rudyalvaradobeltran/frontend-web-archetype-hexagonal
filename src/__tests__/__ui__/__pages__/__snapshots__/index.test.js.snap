// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PAGES Should render snapshot without crashing 1`] = `
<div
  className="sc-gsDJrp hjRpwD"
/>
`;
